### PR TITLE
Implement github actions

### PR DESCRIPTION
**Pull request by alexisbcc**
In Twilio it has been deactivated Travis CI on the GitHub organization due to security issues. This PR implements GitHub Actions. Also some ESLint errors and warnings were fixed.

Windows was removed of the os matrix because the npm script can run only in unix based systems with the current configuration.

Changes:

- Create YAML for GitHub actions job
- Deleted Travis .yml file
- Update README
- Update yarn.lock file
- Add .env files support with `dotenv-safe` package
- Update npm script
- Fix styling issues

### CODE REVIEW

**Review by lopenchi — COMMENTED** [review]
@alexisbcc Good job. Some things I realize: 
- This repo uses yarn (yarn.lock), so it is not needed to create a package.lock file
- In travis.yml node versions were 7, 8. In github actions it is 14, maybe it is a good idea to add in README which version install. What do you think ?

**Comment by alexisbcc** [comment]
> @alexisbcc Good job. Some things I realize:
> 
> * This repo uses yarn (yarn.lock), so it is not needed to create a package.lock file
> * In travis.yml node versions were 7, 8. In github actions it is 14, maybe it is a good idea to add in README which version install. What do you think ?

Thank you I missed that.
Ok I'll add that to the `README`

File: .gitignore [hunk 1]
```diff
@@ -34,5 +34,6 @@ node_modules
 
 #local env file
 .env.local
```
Comment (lopenchi) [inline .gitignore]: I think this line can be deleted because README changed, and this file is not created anymore

**Review by lopenchi — APPROVED** [review]
LGTM

**Comment by jefflinwood** [comment]
Thank you for updating this repo!

**Pull request merged**